### ISSUE TITLE
false Positive: untranslated_string

### ISSUE THREAD

**Issue by blandford** [bilelmoussaoui/gobject-lint#53]
There's no reason to translate empty strings.

I'm getting a hit on:

```C
  self->game_won_toast = adw_toast_new ("");
```

I also got a similar warning for `gtk_label_set_label (" ")`; which was done to create blank label as a spacer.

**Comment by bilelmoussaoui** [comment]
For the first one, it is fixed already by https://github.com/bilelmoussaoui/goblint/commit/831b9ab062045f055ad576b716c18c264a76601d no?

**Comment by bilelmoussaoui** [comment]
actually it isn't... nvm
